src/jwplayer/players/MobileArticleVideoPlayer/Attribution.tsx

```diff
@@ -14,7 +14,7 @@ const Attribution: React.FC = () => {
 			<a className={styles.usernameLink} href={playlistListItem.userUrl}>
 				{playlistListItem.username}
 			</a>
-			<a className={styles.iconLink} href={playlistListItem.userUrl}>
+			<a aria-label={'Attribution Link'} className={styles.iconLink} href={playlistListItem.userUrl}>
```

Comment:
I'd set `aria-label` to `playlistListItem.username` or if we want to have something more descriptive we should apply i18n

Reply:
Yeah that's a good idea